src/views/AddPriceSingle.vue

```diff
@@ -25,7 +25,7 @@
               </v-item-group>
             </h3>
             <v-sheet v-if="productMode === 'barcode'">
-              <v-btn class="mb-2" size="small" prepend-icon="mdi-barcode-scan" @click="showBarcodeScanner">Scan a barcode</v-btn>
+              <v-btn class="mb-2 mt-1" size="small" prepend-icon="mdi-barcode-scan" @click="showBarcodeScanner">Scan a barcode</v-btn>
```

Comment:
mt = margin-top ([docs](https://vuetifyjs.com/en/styles/spacing/#how-it-works))
did you mean `mr` ?

Reply:
has this been fixed?

Reply:
@raphodn I think @duhow wants to add margin top to make space between "Barcode" / "Category" buttons and "Scan a Barcode button"